content/lxd/introduction.md

```diff
@@ -1,87 +1,116 @@
+## What is a system container?
+When running a system container, LXD simulates a virtual version of a full operating system. To do this, it uses the functionality provided by the kernel running on the host system.
 
-# Integration with OpenNebula
-Starting from [OpenNebula EDGE](https://opennebula.org/get-your-hands-on-v-5-8-edge/) this cloud management platform packs official drivers to manage LXD compute nodes.
+### Application containers vs. system containers
+Application containers (as provided by, for example, Docker or Kubernetes) package a single process or application. System containers, on the other hand, simulate a full operating system and let you run multiple processes at the same time.
 
-Key features:
+Therefore, application containers are suitable to provide separate components, while system containers provide a full solution of libraries, applications, databases and so on. In addition, you can use system containers to create different user spaces and isolate all processes belonging to each user space, which is something you cannot do with application containers.
 
+![Application and system containers](/static/img/lxd/application-vs-system-containers.svg "Application and system containers")
 
+### Virtual machines vs. system containers
+Virtual machines emulate a physical machine, using the hardware of the host system from a full and completely isolated operating system. System containers, on the other hand, use the OS kernel of the host system instead of creating their own environment. If you run several system containers, they all share the same kernel, which makes them faster and more light-weight than virtual machines.
 
-* Enables **qcow2 backed containers** and regular KVM-like images, giving users more flexibility when deploying their workloads
-* Support for snap installed LXD nodes.
-* Ceph storage pools shared across all virtualization nodes (both KVM and LXD)
-* Seamless LXD public image servers integration in a marketplace-like style. Supports [LXD public image server](https://images.linuxcontainers.org),  [TurnkeyLinux](https://www.turnkeylinux.org), [DockerHub](https://hub.docker.com/) and also the [OpenNebula marketplace](https://marketplace.opennebula.systems/appliance)
-* [Multi-container services deployment](https://docs.opennebula.org/5.8/advanced_components/application_flow_and_auto-scaling/overview.html) with auto-scaling.
-* Heterogeneous hypervisors cloud deployments with KVM, LXD and VMWare vCenter managed by the same cloud orchestration system
+With LXD, you can create both system containers and virtual machines. You should use a system container to leverage the smaller size and increased performance if all functionality you require is compatible with the kernel of your host operating system. If you need a fully isolated environment or functionality that is not supported by the OS kernel of your host system, use a virtual machine.
```

Comment:
Wouldn't mention MacOS as that doesn't actually work and would be against macOS licensing anyway.

But it may be worth tweaking the last sentence a tiny bit to mention running another OS. I'd actually stay away from "fully isolated environment" as that sounds like VMs would be safer than unprivileged containers and that's mostly up to debate :)

So maybe:
```
If you need functionality that is not supported by the OS kernel of your host system or need to run a completely different OS, use a virtual machine.
```

Reply:
Ok, I would choose "Linux kernel" instead of "OS kernel" though.